### PR TITLE
Issue#196

### LINKED ISSUE #196: Create object for SecureSet in code_schools.yaml
# Feature
## Why is this feature being added?
SecureSet is an amazing partner, and also a Cyber Security Bootcamp. They've requested to be listed on our /code_schools page.

## What should your feature do?
Add SecureSet to ./config/code_schools.yaml in observation of rules set out in the top of that file.

**VA-Approved** 
**Notes:** Not a traditional coding bootcamp. SecureSet focuses on security training relevant to the IT industry.
**Locations:** 
- Colorado Springs, CO 
- Denver, CO 
**Website:** https://secureset.com/

### PR DESCRIPTION
**Pull request by unmonk**
# Description of changes
Adds SecureSet to list of academies 

# Issue Resolved
Fixes #196

### CODE REVIEW

**Comment by kylemh** [comment]
No, making it now!

**Comment by unmonk** [comment]
Is that what is failing?  Reading through the travis log is a nightmare.

**Comment by kylemh** [comment]
@ohaiwalt @hpjaj - I can't tell what's failing in CI.

**Comment by hpjaj** [comment]
@kylemh -

```
Error:
Api::V1::CodeSchoolsControllerTest#test_:index_endpoint_returns_a_JSON_list_of_all_CodeSchools:
Psych::SyntaxError: (/app/config/code_schools.yml): did not find expected '-' indicator while parsing a block collection at line 816 column 5
    app/lib/code_schools.rb:3:in `all'
    app/controllers/api/v1/code_schools_controller.rb:5:in `index'
    test/controllers/api/v1/code_schools_controller_test.rb:22:in `block in <class:CodeSchoolsControllerTest>'
```

**Comment by unmonk** [comment]
Thats the first location listing.   Probably formatting issues, Ill let someone who uses ruby handle this. I might have a different codestyle

File: config/code_schools.yml [hunk 1]
```diff
@@ -802,3 +802,29 @@
...
+      address1: 2228 Blake Street Suite 100
+      city: Denver
+      state: CO
+      zip: 80205
+     - va_accepted: true
+      address1: 523 S Cascade Ave Suite 140
+      city: Colorado Springs
+      state: CO
+      zip: 80903
+    - va_accepted: true
```
Comment (kylemh) [inline config/code_schools.yml]: Change this to false please!

**Comment by hpjaj** [comment]
@unmonk - The indenting here:  https://github.com/unmonk/operationcode_backend/blob/f0e6d7f0a34e11f085f1b43a445fbb03ecb86e9c/config/code_schools.yml#L821-L830

Needs to be corrected to:

```
- name: SecureSet
  url: https://secureset.com
  logo: https://raw.githubusercontent.com/OperationCode/operationcode_frontend/master/src/images/codeSchoolLogos/secureset.jpg
  full_time: true
  hardware_included: false
  has_online: false
  online_only: false
  notes: Not a traditional coding bootcamp. SecureSet focuses on security training relevant to the IT industry.
  email: academy@secureset.com
  locations:
    - va_accepted: true
      address1: 2228 Blake Street Suite 100
      city: Denver
      state: CO
      zip: 80205
    - va_accepted: true
      address1: 523 S Cascade Ave Suite 140
      city: Colorado Springs
      state: CO
      zip: 80903
    - va_accepted: true
      address1: 1915 N. Ave Republica de Cuba
      city: Tampa Bay
      state: FL
      zip: 33605
```

**Pull request closed without merging**